kanin/src/app/task.rs

```diff
@@ -25,6 +25,7 @@ use super::StateMap;
 pub(super) type HandlerTask = Pin<Box<dyn Future<Output = String> + Send>>;
 
 /// Creates the handler task for the given handler and routing key. See [`HandlerTask`].
+#[tracing::instrument(skip_all, field(routing_key = routing_key))]
```

Comment:
Didn't see this before. Won't this add yet another layer of a span? How does it look when printed? Is it too much?

Reply:
oh! I think this was just me debugging/exploring. It doesn't do anything because.. yeah.. I'm not sure tbh.